tasks/notify.py

```diff
@@ -38,6 +41,130 @@
 CUMULATIVE_LENGTH = 10
 
 
+class ExecutionsJobInfo:
+    def __init__(self, job_id: int, failing: bool = True, commit: str = None):
+        self.job_id = job_id
+        self.failing = failing
+        self.commit = commit
+
+    def url(self):
+        return f'{BASE_URL}/DataDog/datadog-agent/-/jobs/{self.job_id}'
+
+    def to_json(self):
+        return {"id": self.job_id, "failing": self.failing, "commit": self.commit}
```

Comment:
The name is misleading, I initially thought it would return a string with the json, here you return a dict. Why not `to_dict`?